### ISSUE TITLE
LightGBM Ranker: How to set the "label gain" parameter? No code examples found!

### ISSUE THREAD

**Issue by sofiavlachou28** [microsoft/LightGBM#5297]

**Comment by nblthree** [comment]
Did you find out how to set it when having multi groups? just in case it can be set like this `model = LGBMRanker(**lgbm_params, label_gain=group.values)`

I am grouping by date like this `group = train_X.groupby('Date').size()`

then I predict like so `model.fit(train_X.values, train_y, group=group.values)`.

Thanks

**Comment by jameslamb** [comment]
I'm locking discussion on this issue, since it overlaps significantly with another issue and I want to keep the discussion focused in one place.

See https://github.com/microsoft/LightGBM/issues/4808#issuecomment-1219044835 for details.
